docs/internal/github_actions.rst

```diff
@@ -0,0 +1,203 @@
+##############################
+GitHub Actions Summary and FAQ
+##############################
+
+********
+Overview
+********
+
+GitHub Actions is the continuous integration solution currently being
+used to evaluate the readiness of pull requests. It builds OpenDDS and runs the
+test suite across a wide variety of operation systems and build configurations.
+
+*****************
+Table of Contents
+*****************
+
+* [Legend for GitHub Actions Build Names]
+  * [Operating System]
+  * [Build Configuration]
+  * [Build Type]
+  * [Build Options]
+  * [Feature Mask]
+* [Introduction to GitHub Actions Workflows]
+* [Caching]
+* [Job Types]
+* [Test Results]
+* [Artifacts]
+  * [Using Artifacts to Replicate Builds]
+  * [Using Artifacts to View More Test Information]
+
+*************************************
+Legend for GitHub Actions Build Names
+*************************************
+
+Operating System
+================
+
+* u18/u20 - Ubuntu 18.04/Ubuntu 20.04
+* w16/w19 - Windows 2016/Windows 2019
+* m10 - MacOS 10.15
+
+Build Configuration
+===================
+
+* x86 - Windows 32 bit. If not specified, x64 is implied.
+* re - Release build.  If not specified, Debug is implied.
+* clang5/clang10/gcc6/gcc8/gcc10 - compiler used to build
+OpenDDS. If not specified, the default system compiler is used.
+
+Build Type
+==========
+
+* stat - Static build
+* bsafe/esafe - Base Safety/Extended Safety build
+* sec - Security build
+* asan - Address Sanitizer build
+
+Build Options
+=============
+
+* o1 - enables --optimize
```

Comment:
```suggestion
* o1 - enables ``--optimize``
```

Literal things like paths and command line options should be marked as monospace to distinguish them.